patches/chrome-browser-resources-side_panel-bookmarks-power_bookmarks_list.ts.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
+index 897919f0015c30e247d10872ae031dc059f413f0..e1e3f74ead610b18919dcf75fdc4ab400cf38a34 100644
+--- a/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
++++ b/chrome/browser/resources/side_panel/bookmarks/power_bookmarks_list.ts
+@@ -171,7 +171,8 @@ export class PowerBookmarksListElement extends PolymerElement {
+                sortOrder: SortOrder.kReverseAlphabetical,
+                label: loadTimeData.getString('sortReverseAlphabetically'),
+                lowerLabel: loadTimeData.getString('sortReverseAlphabetically'),
+-             }],
++             },
++             { sortOrder: SortOrder.kCustom, label: loadTimeData.getString('sortCustom'), lowerLabel: loadTimeData.getString('sortCustomLower'), }],
+       },
+ 
+       editing_: {
+@@ -412,6 +413,7 @@ export class PowerBookmarksListElement extends PolymerElement {
+     const shouldShow = this.bookmarkShouldShow_(bookmark);
+     const isShowing = this.bookmarkIsShowing_(bookmark);
+     if (oldParent === newParent && shouldShow) {
++      if (this.activeSortIndex_ == /*customOrder*/5) { this.updateDisplayLists_(); }
```

Comment:
Upstream doesn't consider UI refresh after moving item in the same parent as that moving doesn't change current view.

Reply:
I think this can be done with a polymer override:

```ts

import { RegisterPolymerPrototypeModification } from 'chrome://resources/brave/polymer_overriding.js'

RegisterPolymerPrototypeModification({
    'power-bookmarks-list': prototype => {
        const old = prototype.onBookmarkMoved
        prototype.onBookmarkMoved = function(bookmark, oldParent, newParent) {
            old(bookmark, oldParent, newParent)
            if (oldParent === newParent && prototype.bookmarkShouldShow_.apply(this, bookmark)) {
                prototype.updateDisplayLists_.apply(this)
            }
        }
    }
})

```

(I might've got the syntax a little wrong 'cause I'm typing into a Github comment).